README.md

```diff
@@ -102,8 +102,9 @@ Dependencies (libraries must be i686):
 - curl with GNUTLS support
 
 Instructions:
+0. Make sure the path leading up the `team-comtress-2` has no spaces in it. This will lead to errors compiling protobud and gperf.

```

Comment:
Wait why? Can we patch this instead?

Reply:
This looks to be cause by improperly quoted `sed` commands:
```bash
$ pwd
.../t c2/team-comtress-2
$ make
...
ar: `u' modifier ignored since `D' is the default (see `U')
libtool: link: ranlib .libs/libprotoc.a
/usr/bin/sed: can't read c2/team-comtress-2/thirdparty/protobuf-2.5.0/src/libprotobuf.la: No such file or directory
libtool: link: `c2/team-comtress-2/thirdparty/protobuf-2.5.0/src/libprotobuf.la' is not a valid libtool archive
make[2]: *** [Makefile:1393: libprotoc.la] Error 1
```

Reply:
See also https://bugs.freedesktop.org/show_bug.cgi?id=31110 and https://lists.gnu.org/archive/html/bug-automake/2007-10/msg00005.html

Reply:
Too much work would be needed to fix this, as a lot of scripts (Makefiles, libtool, protoc, etc.) have broken quotes and other space-related issues.